### PR TITLE
Add Text Highlight component

### PR DESCRIPTION
**Pull request by nathan-de-pachtere**

### CODE REVIEW

File: components/content/inspira/ui/text-highlight/TextHighlight.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+interface Props {
+  text: string;
+  class: HTMLAttributes['class'];
+}
+
+const props = defineProps<Props>();
+</script>
+
+<style scoped>
+@keyframes backgroundExpand {
```
Comment (rahul-vashishtha) [inline components/content/inspira/ui/text-highlight/TextHighlight.vue:21]: Can we use snake case `background-expand`, to keep it consistent across the components

File: components/content/inspira/ui/text-highlight/TextHighlight.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  delay?: number;
+  duration?: number;
+  class?: HTMLAttributes["class"];
+  textEndColor?: string;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  delay: 0,
+  duration: 2000,
+  class: undefined,
```
Comment (rahul-vashishtha) [inline components/content/inspira/ui/text-highlight/TextHighlight.vue:19]: No need to explicitly set the class `undefined`, it is undefined by default

File: content/2.components/text-highlight.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+| `text-end-color` | `string` | `inherit` | Color of the text at the end of the animation. Match WCAG recommendations. |
+
+## Component Code
+
+You can copy and paste the following code to create this component:
+
+::CodeViewer{filename="TextHighlight.vue" language="vue" componentName="TextHighlight" type="ui" id="text-highlight"}
+::
+
+## CREDITS
```
Comment (rahul-vashishtha) [inline content/2.components/text-highlight.md:49]: Keep it normal case

**Review by rahul-vashishtha — CHANGES_REQUESTED** [review]

File: components/content/inspira/ui/text-highlight/TextHighlight.vue (hunk 2 above)
Comment (nathan-de-pachtere) [reply components/content/inspira/ui/text-highlight/TextHighlight.vue:19]: Maybe we should set up ESLint on the repo ?
https://eslint.vuejs.org/rules/require-default-prop.html (and disable this rules if we don't want it 😅 )

Removing `class: undefined` for this PR anyway.

**Review by rahul-vashishtha — APPROVED** [review]

**Pull request merged**